Tools/_sunrise/changelog/changelog_targets.py

```
@@ -0,0 +1,116 @@
+def resolve_target(repo_root: Path, target_id: str) -> ChangelogTarget:
+    target_id = validate_target_id(target_id)
+    for target_path in target_paths(repo_root):
+        if target_path.stem == target_id:
+            return load_target(target_path, repo_root)
+    raise RuntimeError(f"Цель чейнджлога не найдена: {target_id}")
+
+
+def write_github_outputs(target: ChangelogTarget) -> None:
+    output_path = os.environ.get("GITHUB_OUTPUT")
+    if not output_path:
+        raise RuntimeError("Переменная GITHUB_OUTPUT не задана")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)